### DISCUSSION TITLE
zh/docs/provider/oauth/weCom

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3110] [Docs comments]
# zh/docs/provider/oauth/weCom

使用WeCom Syncer同步数据库

https://casdoor.org/zh/docs/provider/oauth/weCom

**Comment by yhgroup** [comment]
看了企业微信的GetToken每次都会调用，没有采用缓存过期前才调

**Comment by yuliqi** [comment]
请问如何获取 当前登录用户的邮箱
